debian/control

```diff
@@ -105,7 +106,7 @@ Build-Depends: automake,
 # Make-Check   xmlstarlet,
                nasm [amd64],
                zlib1g-dev,
-Built-Using:   libyaml-cpp-dev (>= 0.6),
+# Jaeger       Built-Using:   libyaml-cpp-dev (>= 0.6),
```

Comment:
@ideepika  this change was introduced 1e6e1f907eefbc213e297619e9b63df0bd54870d. and i failed to find any explanation in the commit message. would have been great if you could have accompanied it with an explanation why `Built-Using` was used here. but since this fix address the FTBFS, and strictly speaking, `Built-Using` was not added in this change. i am approving it.

Reply:
yes, sorry didn't register that. I will update the explanation, in later commits, thanks for the feedback!